public/index.html

```diff
@@ -15,6 +15,13 @@
 </style>
 <html>
   <body>
+    <h2>Current status</h2>
+    <h3>
+      No. of nodes: <span id="nodes_number"></span><br>
+      Last update: <span id="last_update"></span><br>
+      Next refresh: <span id="refresh_interval"></span><br>
+      Requests used: <span id="requests_used"></span><br>
+    </h3>
     <svg width="1280" height="720"></svg>
```

Comment:
I meant full screen not as enlarging the browser window itself, but making the SVG element in the HTML fill the available space by itself (`height: 100%; width: 100%` rings any bells?) directly at runtime. My 3200x1800 screen might be filled by the current hardcoded proportions, but someone running a smaller resolution might have to scroll the page - which is **not nice**. Same goes for bigger resolutions, they would just see a corner of the page being occupied.

As for _breaking D3_ I meant that most of the solution I've tried make it go berserk and break - not showing any points, showing all of them in a corner, crashing the page itself by outputting errors at a remarkable frequency...